attention_and_transformers.ipynb

```diff
@@ -0,0 +1,4395 @@
+{
```

Comment:
<p>Question: shouldn't they set with<code>_bias=False</code> as well? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>I also wondered a bit about this, but looking at a few implementations and papers (including a new paper https://arxiv.org/pdf/2207.09238.pdf), we keep the bias in here. </p>